pkg/health/checker/checker.go

```diff
@@ -142,54 +177,156 @@ func NewHealthChecker(cClient consulutil.ConsulClient) HealthChecker {
 	}
 }
 
+func (h shadowTrafficHealthChecker) WatchPodOnNode(
+	ctx context.Context,
+	nodeID types.NodeName,
+	podID types.PodID,
+	useHealthService bool,
+	useOnlyHealthService bool,
+	status manifest.StatusStanza,
+) (chan health.Result, chan error) {
+	resultCh := make(chan health.Result)
+	errCh := make(chan error)
+	if !useOnlyHealthService && !h.useOnlyHealthService {
+		resultCh, errCh = consulWatchPodOnNode(ctx, nodeID, podID, h.kv)
+	}
```

Comment:
Am I understanding correctly that this function will not send any traffic to the health service unless it is used exclusively? This is not the desired outcome. Instead, the shadow health checker should send maintain health requests at the health service so we can *observe how the health service scales*.

Reply:
That function will send traffic to the health service if either useHealthService or useOnlyHealthService is true. Health checks to consul will be enabled unless useOnlyHealthService is true.

Reply:
Only one? I see, so that if we are relying on consul data then we shouldn't accept data from the health service. We should only be testing the load against the health service and throwing out the data we receive.